chat-android/src/main/java/com/ably/chat/Typing.kt

```diff
@@ -50,13 +63,13 @@ interface Typing : EmitsDiscontinuities {
      * once the timer expires, a typingStopped event will be emitted. The timeout is configurable through the typingTimeoutMs parameter.
      * If the current user is already typing, it will reset the timer and being counting down again without emitting a new event.
      */
-    suspend fun start()
+    fun start()
```

Comment:
This also means as per [chat-js typing public API spec](https://github.com/ably/ably-chat-js/?tab=readme-ov-file#start-typing), the user should be the one to handle errors emerging when trying to `presence enter clientId` -> https://github.com/ably/ably-chat-js/blob/4dd57bf38f2c333d701186748863a1b8d855a4bc/src/core/typing.ts#L203.

Reply:
Unfortunately, this will need marking the method as `suspend` again : |. 
And we explicitly `.join()` the job inside both `start` and `stop` methods.

Reply:
agree, probably it's better to propagate errors here, will change